htdocs/core/class/html.formpropal.class.php

```diff
@@ -136,4 +136,62 @@ public function selectProposalStatus($selected = '', $short = 0, $excludedraft =
         }
         print '</select>';
     }
+
+	/**
+	 *    Return combo list of all commercial of proposals
+	 *
+	 *    @param	string	$selected   	Preselected value
+	 *    @param    string  $htmlname       Name of select field
+	 *    @param	int		$short			Use short labels
+	 *    @param	int		$excludedraft	0=All status, 1=Exclude draft status
+	 *    @param	int 	$showempty		1=Add empty line
+	 *    @param    string  $mode           'customer', 'supplier'
+	 *    @return	void
+	 */
+	public function selectProposalCommercial($selected = '', $htmlname = 'propal_commercial', $short = 0, $excludedraft = 0, $showempty = 1, $mode = 'customer')
+	{
+		global $langs;
+
+		$prefix = '';
+		$prefix = "PropalStatus";
+
+		$sql = "SELECT rowid, lastname, firstname FROM " . MAIN_DB_PREFIX . "user";
+		$sql .= " WHERE rowid IN";
+		$sql .= " (SELECT commercial FROM ".MAIN_DB_PREFIX."propal_extrafields) ORDER BY firstname ASC";
```

Comment:
This is not possible.
The table propal_extrafields is empty by default and does not contains any column "commercial". This table contains custom columns with name that may depends on each Dolibarr installation, so having a select with a hard coded column  name will return a SQL syntax error here on most Dolibarr installation.

Reply:
I recommend to make 2 separate PR, 1 for each added feature so the second one (use of date of signature) will not be blocked by the other one.